components/dropdown/dropdown-button.js

```diff
@@ -25,6 +25,7 @@ class DropdownButton extends DropdownOpenerMixin(RtlMixin(LitElement)) {
 				height: 0.8rem;
 				width: 0.8rem;
 				margin-left: 0.6rem;
+				pointer-events: none;

```

Comment:
Curious - what prompted this?

Reply:
That was me. I noticed that in the LMS clicking the dropdown text or icon wouldn't cause focus styles to be applied, but clicking elsewhere would. Things are slightly different in the Lit version, where only clicking the icon would cause it.